src/core_plugins/kibana/public/dashboard/dashboard_app.html

```diff
@@ -46,22 +51,66 @@
     class="dshStartScreen"
   >
-    <h2 class="kuiTitle kuiVerticalRhythm">
-      This dashboard is empty. Let&rsquo;s fill it up!
+    <h2
+      class="kuiTitle kuiVerticalRhythm"
+      i18n-id="kbn.dashboard.fillDashboardTitle"
+      i18n-default-message="This dashboard is empty. Let&rsquo;s fill it up!"
+    >
     </h2>
 
-    <p class="kuiText kuiVerticalRhythm">
-      Click the <a kbn-accessible-click class="kuiButton kuiButton--primary kuiButton--small" ng-click="showAddPanel()" aria-label="Add visualization" data-test-subj="emptyDashboardAddPanelButton">Add</a> button in the menu bar above to add a visualization to the dashboard. <br/>If you haven't set up any visualizations yet, <a class="kuiLink" href="#/visualize">visit the Visualize app</a> to create your first visualization.
+    <p>
+      <span
+        i18n-id="kbn.dashboard.addVisualizationDescription1"
+        i18n-default-message="Click the "
+        i18n-context="Part of composite label kbn.dashboard.addVisualizationDescription1 + {link} + kbn.dashboard.addVisualizationDescription2"
+      ></span>
+      <a
+        kbn-accessible-click
+        class="kuiButton kuiButton--primary kuiButton--small"
+        ng-click="showAddPanel()"
+        aria-label="{{ 'kbn.dashboard.addVisualizationLinkAriaLabel' | i18n: { defaultMessage: 'Add visualization' } }}"
+        data-test-subj="emptyDashboardAddPanelButton"
+        i18n-id="kbn.dashboard.addVisualizationLinkText"
+        i18n-default-message="Add"
+      ></a>
+      <span
+        i18n-id="kbn.dashboard.addVisualizationDescription2"
+        i18n-default-message=" button in the menu bar above to add a visualization to the dashboard. {br}If you haven't set up any visualizations yet, {visitVisualizeAppLink} to create your first visualization."
+        i18n-values="{
+          br: '<br/>',
+          visitVisualizeAppLink: '<a class=\'kuiLink\' href=\'#/visualize\'>' + visitVisualizeAppLinkText + '</a>'
+        }"
+      ></span>
     </p>
   </div>
 
   <div ng-show="getShouldShowViewHelp()" class="dshStartScreen">
-    <h2 class="kuiTitle kuiVerticalRhythm">
-      This dashboard is empty. Let&rsquo;s fill it up!
+    <h2
+      class="kuiTitle kuiVerticalRhythm"
+      i18n-id="kbn.dashboard.fillDashboardTitle"
+      i18n-default-message="This dashboard is empty. Let&rsquo;s fill it up!"
+    >
     </h2>
 
-    <p class="kuiText kuiVerticalRhythm">
-      Click the <a kbn-accessible-click class="kuiButton kuiButton--primary kuiButton--small" ng-click="enterEditMode()">Edit</a> button in the menu bar above to start working on your new dashboard.
+    <p
+      class="kuiText kuiVerticalRhythm"
+    >
+      <span
+        i18n-id="kbn.dashboard.howToStartWorkingOnNewDashboardDescription1"
+        i18n-default-message="Click the "
+        i18n-context="Part of composite label kbn.dashboard.howToStartWorkingOnNewDashboardDescription1 + {link} + kbn.dashboard.howToStartWorkingOnNewDashboardDescription2"
```

Comment:
I think it would be clearer for vendor:
```suggestion
        i18n-context="Part of composite label kbn.dashboard.howToStartWorkingOnNewDashboardDescription1 + {link: kbn.dashboard.howToStartWorkingOnNewDashboardEditLinkText} + kbn.dashboard.howToStartWorkingOnNewDashboardDescription2"
```